### DISCUSSION TITLE
Setup HomeAssistant-OctopusEnergy

### DISCUSSION THREAD

**Discussion by yebaws** [BottlecapDave/HomeAssistant-OctopusEnergy#276] [General]
Please bear in mind that I'm completely new to HA....

Have followed instructions for setting up HomeAssistant-OctopusEnergy by HACS.

Successfully added repository, downloaded the files and then restarted HA. Had expected to see OctopusEnergy appear in my integrations dashboard, but there is nothing there.

If I go to HACS >> Integrations I find OctopusEnergy there, but clicking on it just takes me to the setup guide.

What blindingly obvious thing am I missing here?

**Comment by BottlecapDave** [comment]
Hi there.

It sounds like you've installed the integration, but not yet configured it. You can do this by following theses [instructions](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/setup_account.md#setup-account)

**Reply by yebaws** [reply]
Thank you! was hoping it would be that simple.
